tests/test_RoseTree.ml

```diff
@@ -0,0 +1,600 @@
+open Containers_misc
+open OUnit
+
+let format_node value = Printf.sprintf "%d" value
+
+let string_of_tree tree =
+  let buffer = Buffer.create 1024 in
+  let formatter = Format.formatter_of_buffer buffer in
+  RoseTree.print formatter format_node tree;
+  Buffer.contents buffer
```

Comment:
How so? `CCFormat.sprintf` looks like it takes a formatting string and some arguments, but `RoseTree.print` (in its current state) needs a `formatter` (I suppose I could have it accept a `CCFormat.printer` though).